### ISSUE TITLE
WindowComponent: cannot update width/height property

### ISSUE THREAD

**Issue by cluetjen** [telerik/kendo-angular#1683]
## I'm submitting a...
* Bug report 

( bug / feature request depends on personal point of view, but it seems to be quite easy to fix / extend)

## Current behavior

* Open new window (using window service)
* Update win.width property
* => Width of window will not change

(same for height property)

## Expected behavior

Window should have new width.

https://www.telerik.com/kendo-angular-ui/components/dialogs/api/WindowComponent/

> width number
> Specifies the width of the Window. The width property has to be set in pixels.

## Minimal reproduction of the problem with instructions
https://plnkr.co/edit/lGpgbmONzCzC0GVved3g?p=preview

or..
    
    const win = this._windowRef.window.instance;
    win.width = 800;
    // => nothing happens
    win['updateAllOffset']();
    // now it works, but that's a private method

## Environment

Package versions:

```
"@progress/kendo-angular-dialog@^3.6.2":
  version "3.6.2"
  resolved "https://registry.yarnpkg.com/@progress/kendo-angular-dialog/-/kendo-angular-dialog-3.6.2.tgz#afea442a7f8ba19529d93b7fc2b935f812921c46"
  dependencies:
    "@progress/kendo-popup-common" "^1.2.3"
    "@progress/kendo-schematics" "^0.2.0"
    "@telerik/kendo-draggable" "^1.5.1"
    tslib "^1.7.0"
```

## Remarks

Example use case:

We have a window where content is generated dynamically and can be modified by users so ...
* It's difficult to specify width / height when opening the window (we simply don't know it)
* If we let the window render without, size will be perfect but as soon as the user modifies the inner content, window size will change (nooo,... that doesn't look good)...
* => We could simply let the window render, get dimensions and then set width / height and everything will work.. - but: no getSize(), el is private and no resize() and width/height not working :-(

**Comment by Raisolution** [comment]
Hi @cluetjen,

Thank you for pointing this out. We will consider to expose public API methods for dynamically created Window components via the service.

Regarding the declaratively created components, you could utilize two-way binding for the width/height properties in order to update the Window dimensions dynamically. This is demonstrated on the [following documentation page](https://www.telerik.com/kendo-angular-ui/components/dialogs/window/dimensions-resizing/). At the moment I would recommend to use initially hidden declarative Window with two-way bound width/height.

**Comment by cluetjen** [comment]
Hi @Raisolution,

thanks - I see. When using the service, we've no bindings, so no ngOnChanges ... In our case working with declarative windows is problematic because there's no "logical parent component". Windows can be opened from multiple places. It's encapsulated in a custom service, so I'll stay with calling the private functions for now - should be easy to fix when your internal api changes...

**Comment by Raisolution** [comment]
We introduced the new `setDimension` method for scenarios like this one. It is available in `v3.7.0-dev.201808011212` and also will be present in the next official release.

**Comment by Raisolution** [comment]
`v3.7.0` has been released.
